swarm/storage/netstore.go

```diff
@@ -121,7 +121,10 @@ func (self *NetStore) Get(key Key) (chunk *Chunk, err error) {
 	}
 }
 
-func (self *NetStore) get(key Key) (chunk *Chunk, err error) {
+func (self *NetStore) get(key Key, timeout time.Duration) (chunk *Chunk, err error) {
+	if timeout == 0 {
```

Comment:
This is not obvious behaviour, I think we should be explicit with the params. We are sending `defaultRetrieveTimeout` everywhere this is used, and we are passing 0 at only one place - I think being explicit would be easier to read and reason about.

Reply:
I am also not sure why we are using two different timeouts for this. Ultimately this should come from a `context.Context` once we refactor this functionality.

Reply:
It should be `context` I agree.